src/EnergyPlus/DataGlobalConstants.cc

```diff
@@ -652,6 +658,146 @@ namespace DataGlobalConstants {
         return ResourceTypeChar;
     }
 
+    void ValidateFuelType(std::string FuelTypeInput)
```

Comment:
@Myoldmopar @mjwitte is DataGlobalConstants the right place for this?

Reply:
I'm not sure.  Maybe UtilityRoutines.cc?

Reply:
I'll move the function to UtilityRoutines.cc!